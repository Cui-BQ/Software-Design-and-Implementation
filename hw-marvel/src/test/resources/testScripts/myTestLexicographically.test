LoadGraph myTest myTest.tsv

#Two paths, one via ABC, other one via AB
FindPath myTest A B

#Two paths, A -> (ABC) -> B -> (BCX) -> X
#        Or A -> (ABC) -> C -> (BCX) -> X
FindPath myTest A X