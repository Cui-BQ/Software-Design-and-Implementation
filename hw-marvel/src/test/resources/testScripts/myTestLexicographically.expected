loaded graph myTest

#Two paths, one via ABC, other one via AB
path from A to B:
A to B via ABC

#Two paths, A -> (ABC) -> B -> (BCX) -> X
#        Or A -> (ABC) -> C -> (BCX) -> X
path from A to X:
A to B via ABC
B to X via BCX